requirements/test.txt

```diff
@@ -1,4 +1,4 @@
 case>=1.3.1
-pytest>=3.0
+pytest>=3.0,<=5.3.5
```

Comment:
<=5.4?

Reply:
With 5.4 pytest-sugar is not working - see: https://github.com/Teemu/pytest-sugar/issues/187